Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -1144,7 +1081,7 @@ - (void)viewDidAppear:(BOOL)animated
     [self updateBackButtonUnreadCount];
     [self autoLoadMoreIfNecessary];
 
-    self.focusMessageIdOnOpen = nil;
+    self.conversationViewModel.focusMessageIdOnOpen = nil;
```

Comment:
In the long run, I think we can achieve a cleaner & simpler relationship between CVC and CVM.  For example, I'd rather that (eventually) CVM not know about `focusMessageIdOnOpen` but instead know about `messageIdToTryToIncludeInLoadWindow` (or whatever).  But I'm trying to take this PR as a first step toward disentangling the CVC and CVM.